src/Normalizer/Indexer/PapCampaignNormalizer.php

```diff
@@ -16,4 +17,33 @@ protected function getTitle(object $object): string
     {
         return $object->getTitle();
     }
+
+    protected function getType(): string
+    {
+        return JeMengageTimelineFeedEnum::PAP_CAMPAIGN;
+    }
+
+    /** @param Campaign $object */
+    protected function getDescription(object $object): ?string
+    {
+        return 'Nouvelle campagne de porte-à-porte autour de vous.';
+    }
+
+    /** @param Campaign $object */
+    protected function isLocal(object $object): bool
+    {
+        return false;
+    }
+
+    /** @param Campaign $object */
+    protected function getDate(object $object): ?\DateTime
+    {
+        return $object->getCreatedAt();
```

Comment:
should it be `getBeginAt()` instead?

Reply:
no, I confirmed it with victor. He wants that if you change the beginAT or finishAT date, the position in the list does not change.